tap_jira/__init__.py

```
@@ -157,6 +158,20 @@ def main_impl():
 def main():
     try:
         main_impl()
+    except requests.exceptions.HTTPError as http_err:
+        # Check if this is the specific 400 error for Jira search API
+        if http_err.response.status_code == 400 and '/rest/api/2/search' in http_err.response.url:
```

@myleshenderson is this the best place to do this?

We spin up multiple threads for multiple jira projects somewhere inside `main_impl`.  If we are getting this error here that means the other threads are still running and the program will finish before they complete successfully.

I think this will lead to stopping execution early as opposed to silently failing.